presto-pinot-toolkit/src/main/java/com/facebook/presto/pinot/query/PinotFilterExpressionConverter.java

```diff
@@ -294,6 +311,10 @@ public PinotExpression visitCall(CallExpression call, Function<VariableReference
                 return handleLogicalBinary(operatorType.getOperator(), call, context);
             }
         }
+        switch (functionMetadata.getName().getFunctionName()) {
```

Comment:
Its a bit weird to have a single case switch.

Reply:
let me change it to string compare